app/routers/user.py

```diff
@@ -1,13 +1,19 @@
+from fastapi import APIRouter, Depends, Request
 from typing import List
+from starlette.responses import RedirectResponse
 from pydantic import BaseModel, Field
 
 from sqlalchemy.exc import SQLAlchemyError
 from sqlalchemy.orm import Session
 
+from starlette.status import HTTP_202_ACCEPTED
+
 from app.dependencies import get_db
 from app.database.models import Event, User, UserEvent
-from app.internal.utils import save
-from fastapi import APIRouter, Depends
+from app.internal.utils import save, get_current_user
+from app.internal.user.availability import (
+    disable, enable)
+
```

Comment:
Sort by abc and split into 3 paragraphs, just as you did in app/internal/user/availability.py

Reply:
fixed :D

Reply:
fixed :D

Reply:
fixed :D